lib/maxrectpacker.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
I think this would serve better as a separate node module that we could introduce as a dependency.